### PR TITLE
T11: Build Signup Form

### PR DESCRIPTION
**Pull request by iamjny**
image

The first PR had a mistake with the 'SIGN UP [AS]' button but has now been fixed.

### CODE REVIEW

File: frontend/src/index.css [hunk 1]
```diff
@@ -5,6 +5,8 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  height: 100vvh;
```
Comment (kdata12) [inline frontend/src/index.css]: small typo here, it should be "100vh"?

File: frontend/src/components/Signup/Signup.jsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+        <form>
+          <label htmlFor="email">EMAIL</label>
+          <input type="text" id="email" name="email" placeholder="Text" />
+          
+          <label htmlFor="password">PASSWORD</label>
+          <input type="password" id="password" name="password" placeholder="Text" />
+
+          <label htmlFor="password">CONFIRM PASSWORD</label>
+          <input type="password" id="password" name="password" placeholder="Text" />
```
Comment (kdata12) [inline frontend/src/components/Signup/Signup.jsx]: The form has two password inputs with the same id attribute (id="password"). What you can do here is to change the confirm password input id to "confirmPassword"

```css
<label htmlFor="passwordConfirm">CONFIRM PASSWORD</label>
<input type="password" id="passwordConfirm" name="passwordConfirm" placeholder="Text" />
```

**Comment by kdata12** [comment]
Overall, this is honestly looking really good! One nit is that the jsx should have proper indentation and spacing for better readability. A way to conveniently format your code is to install the `prettier` vscode extension to auto format your code after every time you save the file.

https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

File: frontend/public/index.html [hunk 3]
```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
```
Comment (justinpchen94) [inline frontend/public/index.html:27]: hahah let's actually replace this with your app name, Permuto!

File: frontend/src/App.js [hunk 4]
```diff
@@ -1,24 +1,11 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <div>
+      <Signup/>
+      </div>
```
Comment (justinpchen94) [inline frontend/src/App.js]: nit: unindent

File: frontend/src/components/Signup/Signup.css [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+  color: #FF4000;
+  font-size: 17px;
+  font-weight: bold;
+}
+
+input {
+  margin-bottom: 20px;
+  padding: 10px;
+  width: 100%;
+  /* Text box border color */
```
Comment (justinpchen94) [inline frontend/src/components/Signup/Signup.css]: this and the 2 comments below are probably not needed long term

File: frontend/src/components/Signup/Signup.jsx [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+
+      </div>
+      
+    </div>
+    
+  )
+}
+
+export default Signup
```
Comment (justinpchen94) [inline frontend/src/components/Signup/Signup.jsx]: new line below pls

File: frontend/src/components/Signup/Signup.jsx [hunk 7]
```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import './Signup.css'
+
+const Signup = () => {
```
Comment (justinpchen94) [inline frontend/src/components/Signup/Signup.jsx]: nit: `SignUp`

**Review by justinpchen94 — COMMENTED** [review]
some small comments!

**Review by kdata12 — APPROVED** [review]
approved

**Pull request merged**